pkgs/development/compilers/rust/rustc.nix

```diff
@@ -97,11 +97,11 @@ stdenv.mkDerivation {
     rm -vr src/test/run-make/linker-output-non-utf8 || true
 
     # Remove test targeted at LLVM 3.9 - https://github.com/rust-lang/rust/issues/36835
-    rm -vr src/test/run-pass/issue-36023.rs || true
+    rm -fvr src/test/run-pass/issue-36023.rs
```

Comment:
IIRC: i think the two changes (the places we added the `rm` are only required for `stable`) and made `beta` and `nightly` fail.